firefox-ios/firefox-ios-tests/Tests/ClientTests/Telemetry/HomepageTelemetryTests.swift

```diff
@@ -11,7 +11,7 @@ final class HomepageTelemetryTests: XCTestCase {
     override func setUp() {
         super.setUp()// Due to changes allow certain custom pings to implement their own opt-out
         // independent of Glean, custom pings may need to be registered manually in
-        // tests in order to puth them in a state in which they can collect data.
+        // tests in order to put them in a state in which they can collect data.
         Glean.shared.registerPings(GleanMetrics.Pings.shared)
         Glean.shared.resetGlean(clearStores: true)
     }
```

Comment:
Just noticed, we should remove those and rewrite the tests `testPrivateModeShortcutToggleTappedInNormalMode` and `testPrivateModeShortcutToggleTappedInPrivateMode` to use a `MockGleanWrapper` as well. 
Basically the unit test class shouldn't have to `import Glean` at all.
```suggestion
    }
```

Reply:
will look onto it.